src/applications/vaos/components/PastAppointmentsList.jsx

```diff
@@ -70,7 +77,9 @@ export class PastAppointmentsList extends React.Component {
                       appointment={appt}
                       facility={
                         systemClinicToFacilityMap[
-                          `${appt.facilityId}_${appt.clinicId}`
+                          `${parseFakeFHIRId(
```

Comment:
Do you need `parseFakeFHIRId` for any of the `getVARFacilityId` calls? It doesn't seem like that function will ever return an id with `var` in front of it.

Reply:
I'm not sure if this will be how the HeathcareService is returned, but in the transformer I  have it set as: 
```
      {
        actor: {
          reference: `HealthcareService/var${appt.facilityId}_${appt.clinicId}`,
          display:
            appt.clinicFriendlyName || appt.vdsAppointments?.[0]?.clinic?.name,
        },
      },
```
`getVARFacilityId()` should return var${appt.facilityId}` so I think we'd need the `parseFakeFHIRId()` right?.

Reply:
Ah, yeah, looks like maybe I read that wrong.